src/main/drivers/pwm_mapping.h

```diff
@@ -85,7 +85,8 @@ typedef enum {
     PWM_PF_SERVO = (1 << 1),
     PWM_PF_MOTOR_MODE_BRUSHED = (1 << 2),
     PWM_PF_OUTPUT_PROTOCOL_PWM = (1 << 3),
-    PWM_PF_OUTPUT_PROTOCOL_ONESHOT = (1 << 4)
+    PWM_PF_OUTPUT_PROTOCOL_ONESHOT = (1 << 4),
+    PWM_PF_OUTPUT_PROTOCOL_DIGITAL = (1 << 5),
```

Comment:
Should now be `...PROTOCOL_DSHOT`, since there may be other digital protocols in future.